README.md

```diff
@@ -27,7 +27,7 @@ There are several ways to setup Raster Vision:
     - We publish a new tag per merge into `master`, which is tagged with the first 7 characters of the commit hash. To use the latest version, pull the `latest` suffix, e.g. `raster-vision:pytorch-latest`. Git tags are also published, with the Github tag name as the Docker tag suffix.
 * Raster Vision can be installed directly using `pip install rastervision`. However, some of its dependencies will have to be installed manually.
 
-For more detailed instructions, see the [Setup docs](https://docs.rastervision.io/en/0.13/setup.html).
+For more detailed instructions, see the [Setup docs](https://docs.rastervision.io/en/0.20-dev/setup.html).
```

Comment:
Why is it 0.20-dev and not just 0.20?

Reply:
I saw [another library doing it this way](https://github.com/sphinx-gallery/sphinx-gallery/blob/master/sphinx_gallery/__init__.py) and thought it made sense. Under this scheme, the version in the master branch would always be `<next planned version>-dev` and then the `-dev` suffix would be removed in the release branch.

Similar ideas are suggested in [this StackOverflow question](https://stackoverflow.com/questions/27185317/versioning-on-development-and-release-branches-git-flow).